README.md

```diff
@@ -1,5 +1,51 @@
-# Mini-Projects
+# Machine Learning Engineering Course Mini-Projects
+
+This is the repository of the current miniprojects for Springboard's Machine Learning Engineering course. As you work through the curriculum, you will need to complete each and everyone of those mini-projects and review and discuss them with your mentor.
+
+It is highly recommended to fork this repository on Github and share it with your mentor so that they can easily follow your progress.
+
+## Installation
+
+Below are details instructions on how to get those mini projects up and running. None of those projects should require tons of CPU or memory, so you are welcome to either run those locally on your computer, or through any hosted jupyter notebook services such as Deepnote or Google's Collab.
+
+### Local
+
+To run those projects locally, we recommend you first install Anaconda for your operating system (Windows, Mac, Linux) by downloading it and following the instructions here: https://www.anaconda.com/products/individual#Downloads
+
+Once anaconda is installed, run the following command to create dedicated python environment with all the libraries and packages needed to run all the mini projects in this repo:
+```conda env create -f environment.yml```
+
+Then activate the environment by running:
+```conda activate mec-miniprojects```
+
+Then create an ipython kernel for Jupyter based on this environment:
+```python -m ipykernel install --user --name mec-miniprojects --display-name "Python (MEC Mini-Projects)"```
+
+Finally, launch Jupyter Lab and navigate to each mini-project notebook and make sure to select the kernel that was created above:
+```jupyter lab```
+
+### Google Colab
+
+It is actually very easy to work and run the notebooks for each mini-projects in Google Colab by leveraging the Github integration. Please note that you will need to have forked the mini-projects repository for this to work.
```

Comment:
@arnaudsj can you also add a note that this only works for mini-projects that are in Jupyter notebooks. Some of the projects are in markdown files and cannot be imported to colab this way. (alternatively we could update those files to just be in notebooks which might be preferable)

Reply:
Got it! Added a small precision, and yes, let's put this on the list to convert this one to a Notebook!